roles/openshift_deploy_registry/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Deploy registry
+  shell: osadm registry --create --credentials=/var/lib/openshift/openshift.local.certificates/openshift-registry/.kubeconfig --images='registry.access.redhat.com/openshift3_beta/ose-${component}:${version}'
```

Comment:
We can use the openshift_registry_url (or rather, in my configTemplates branch, I've refactored openshift_facts a bit, so that it now provides openshift.{master,node}.registry_url which woudl also work here.